command/certificate/verify.go

```diff
@@ -164,6 +177,30 @@ func verifyAction(ctx *cli.Context) error {
 		}
 	}
 
+	if expire {
+
+		NowTillEndOfCert := time.Until(cert.NotAfter)
```

Comment:
variable names with capital first letters are exported. Use a lower case first letter unless you need to export the variable beyond the current package.

Reply:
use `remainingValidity` and `totalValidity`. The term validity window is already associated with certificates and we already frequently use the work `validity` in our code when doing similar calculations.